src/NUnitTestAdapter/NUnitTestAdapter.cs

```diff
@@ -177,7 +179,15 @@ private TestPackage CreateTestPackage(string assemblyName)
 
             // Always run one assembly at a time in process in it's own domain
             package.Settings[PackageSettings.ProcessModel] = "InProcess";
-            package.Settings[PackageSettings.DomainUsage] = "Single";
+
+            if (Settings.DomainUsage != null)
+            {
+                package.Settings[PackageSettings.DomainUsage] = Settings.DomainUsage;
```

Comment:
I believe he checks in AdapterSettings but a test of a bad value would be good if we don't have one already.